### ISSUE TITLE
'ariaLabel' behaviour with multiselect

### ISSUE THREAD

**Issue by ganapathyHf** [cibernox/ember-power-select#1272]
Currently, I believe that the 'ariaLabel' property gets applied on the trigger `div` element. Incase of multi-select with search enabled, the 'tabindex' on the trigger element is -1.

This means that it is the input element inside the trigger element that gets focused upon TAB. If this is the expected behaviour, should the 'ariaLabel' be applied on the input element rather than the trigger `div`?

The above scenario works fine for single select as the trigger `div` is the focusable element there.

**Comment by cibernox** [comment]
@ganapathyHf yes, the tabindex of the input is -1 so the focus flows automatically to the input inside. About wether it would be more correct to apply the ariaLabel to the input i'm not sure, a11y is not my strongest suite, but I wouldn't mind changing it.

**Comment by ganapathyHf** [comment]
@cibernox Thanks for the quick response. 

The thing is that I would expect my screenreader to announce what the input refers to, once it is focused. Hence the question.

**Comment by cibernox** [comment]
Makes sense. Can you confirm that putting the aria-label in the input works better?

**Comment by ganapathyHf** [comment]
@cibernox  Hi, Sorry for the late response.

Yes. aria-label for the <input> element works fine.

**Comment by cibernox** [comment]
@ganapathyHf Cool. I labeled this as a bug and try to implement it, but I first need to rearchitecture the component a bit to, conditionally, pass the `aria-label` (and also the `aria-labelledby`!!!) to the trigger or the input depending if the select is multiple or single.

**Comment by ganapathyHf** [comment]
@cibernox . Cool. And thanks for a great component;)

**Comment by mum-never-proud** [comment]
@cibernox if no one's working, shall i raise a pr?

**Comment by cibernox** [comment]
I'm not sure this can be fixed right now without breaking the component's public API.
